Resources/Private/Fusion/Document/Fragment/Menu/Secondary.fusion

```diff
@@ -0,0 +1,29 @@
+prototype(Neos.Demo:Document.Fragment.Menu.Secondary) < prototype(Neos.Fusion:Component) {
+
+    menuItems = Neos.Neos:MenuItems {
+        entryLevel = 2
+        maximumLevels = 1
+    }
+
```

Comment:
There should be a `@if.hasMenuItems = ${this.menuItems}` here to hide the menu when it's empty.

Reply:
@Sebobo added ` @if.hasMenuItems={props.menuItems} ` to the afx renderer of meta and secondary menu . The other menus are not optional.